### PR TITLE
Ft nags user until resolved #158174590

### PR DESCRIPTION
**Pull request by archbishop12**
What does this PR do?
* Add continues alerts to the user if job remains unresolved
Description of Task to be completed?
Implementing the following changes to the files
* templates/front/my_checks.html
    * Add nag icon 
    * Add nag interval option for user
* front/views.py
    * Add nag interval logic for updating the value
* Api/management/sendalerts.py
    * Add logic to send alerts to user if job goes down until it goes up again.
* Front/tests/test_update_nag_interval.py
    * Testing update_nag_interval method.
* Api/tests/test_send_alerts_resolved.py
    * Testing it handles unresolved.
How should this be manually tested?
After cloning the repo, run the following commands
* python manage.py test

What are the relevant pivotal tracker stories?
#158174590
screen shot 2018-06-28 at 17 22 01

### CODE REVIEW

**Comment by EdwinKato** [comment]
@archbishop12 remove the virtual_env files from the remote repo

**Comment by EdwinKato** [comment]
@archbishop12 your build is failing. Some test is failing

**Comment by archbishop12** [comment]
Hi Edwin, the build is now passing.

**Comment by EdwinKato** [comment]
@archbishop12 Resolve conflicts

**Comment by archbishop12** [comment]
Conflicts resolved

File: hc/api/management/commands/sendalerts.py [hunk 1]
```diff
@@ -17,15 +17,35 @@ class Command(BaseCommand):
...
+        
+        if len(repeat_list_approved) == 1:
+            checks = (list(going_down.iterator()) + list(going_up.iterator()) + list(repeat_list_approved))
+        else:    
+            checks = (list(going_down.iterator()) +
+                    list(going_up.iterator()) + list(repeat_list_approved.iterator()))
+        trial_checks = list(repeat_list_approved.iterator())
+        
         if not checks:
+            print("false")
```
Comment (EdwinKato) [inline hc/api/management/commands/sendalerts.py]: I'm guessing this was left out after debugging. If so remove it after

File: hc/settings.py [hunk 2]
```diff
@@ -145,6 +135,13 @@
 COMPRESS_OFFLINE = True
 
 EMAIL_BACKEND = "djmail.backends.default.EmailBackend"
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_PORT = 587
+EMAIL_HOST_USER = 'hcaceskla@gmail.com'
+EMAIL_HOST_PASSWORD = 'hc@ce5kla'
```
Comment (EdwinKato) [inline hc/settings.py]: Configure sensitive information such as the password in the local settings file or the environment. Committing it with your source code could expose a possible vulnerability

**Review by EdwinKato — CHANGES_REQUESTED** [review]

**Comment by archbishop12** [comment]
I have removed the email credentials and the print.

**Comment by EdwinKato** [comment]
@archbishop12 resolve conflicts

**Review by EdwinKato — APPROVED** [review]

**Pull request merged**